recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "streamlit-feedback" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 90f198a4e116e901aaf08058261d444f0abcd3a365bac8c5d6fc08a3c05e02a2
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # s390x is missing streamlit
+  skip: true  # [py<37 or s390x]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+    - streamlit >=0.63
```

Comment:
```suggestion
```
See https://github.com/trubrics/streamlit-feedback/blob/main/setup.py#L31. Install here means "run".

Reply:
Removed, thank you!